packages/ts/react-crud/src/locale.ts

```diff
@@ -100,9 +98,7 @@ export class LocaleFormatter {
 
     // Verify that the parsed date is valid
     const dateInstance = new Date();
-    dateInstance.setFullYear(year);
-    dateInstance.setMonth(month);
-    dateInstance.setDate(day);
+    dateInstance.setFullYear(year, month, day);
```

Comment:
Could be shortened down to `new Date(year, month, day);` I guess?

```suggestion
      const dateInstance = new Date(year, month, day);
```